qa/L0_s3_local/test.sh

```diff
@@ -44,15 +44,19 @@ CLIENT_LOG="./client.log"
 PERF_CLIENT=../clients/perf_client
 
 DATADIR="/data/inferenceserver/${REPO_VERSION}/qa_model_repository"
+BACKENDS="graphdef libtorch onnx plan savedmodel"
 
-MODELDIR="models"
-rm -rf ./$MODELDIR && mkdir $MODELDIR
-cp -r $DATADIR/*_float32_float32_float32 $MODELDIR/.
+rm -rf models && mkdir models
+for BACKEND in $BACKENDS; do
+    cp -r $DATADIR/${BACKEND}_float32_float32_float32 models/.
+    # Remove version policy from config.pbtxt
```

Comment:
Why is this needed?

Reply:
The version policy allows 1 and 3 but we want to autoload 4 when and unload 3 when 4 is added to model repo and polling is enabled